integrations/google_ai/src/haystack_integrations/components/generators/google_ai/chat/gemini.py

```diff
@@ -266,10 +271,21 @@ def _message_to_content(self, message: ChatMessage) -> Content:
             part.function_response.response = message.text
         elif message.is_from(ChatRole.USER):
             part = self._convert_part(message.text)
+        elif "TOOL" in ChatRole._member_names_ and message.is_from(ChatRole.TOOL):
```

Comment:
(Even if I don't particularly like this check) this is done to check if the message has the tool role - only present in Haystack main (to be released in 2.9.0)